frontend/app/src/features/room/components/info/TopicBadge.tsx

```diff
@@ -0,0 +1,14 @@
+import { Badge } from "@chakra-ui/react"
+
+interface TopicBadgeProps {
+    topic: string;
+}
+
+export const TopicBadge = ({ topic }: TopicBadgeProps) => {
+    return (
+        <Badge colorScheme="gray" variant="outline" textTransform={"capitalize"} fontWeight={"normal"}>
```

Comment:
i don't think `fontWeight` is necessary if it's just normal right or does it default to something else?

Reply:
It defaults to bold